### PR TITLE
feat: Update navigation bar to support Heart

### PR DESCRIPTION
**Pull request by ActuallyACat**
# Objective
Update navigation bar (both the component library and Zen packages) to support Heart. The changes here are: 
* Update to use the new `kz-var` variables
* Change default navigation bar  to be wisteria-600 on Heart, and keep wisteria-700 on Zen
* Change content navigation bar to be cluny-200 on Heart, and keep cluny-100 on Zen
* Add a separator on content and admin variants 

Related stories:
https://dev.cultureamp.design/ally/heart-navigation/storybook/?path=/story/zennavigationbar-react--admin-colours
https://dev.cultureamp.design/ally/heart-navigation/storybook/?path=/story/navigationbar-deprecated-react--content-colors

# Screenshots (if appropriate)
Screen Shot 2021-02-23 at 11 05 56 am
Screen Shot 2021-02-23 at 11 05 47 am
Screen Shot 2021-02-23 at 11 07 18 am
Screen Shot 2021-02-23 at 11 07 10 am
Screen Shot 2021-02-23 at 11 07 58 am
Screen Shot 2021-02-23 at 11 07 50 am

# Checklist

- [x] If this contains visual changes, has it been reviewed by a designer?
- [ ] If this introduces either a new component or a breaking change to an existing component, has it been reviewed by an advocate? (Ask in #prod_design_systems)
- [ ] I have considered likely risks of these changes and got someone else to QA as appropriate
- [ ] I have or will communicate these changes to the front end practice

# Additional Considerations
- Does there need to be right-to-left (RTL) options for localization and internationalization?
- Has the test suite been updated?
- Have you done cross-browser testing for our [supported browsers](https://academy.cultureamp.com/hc/en-us/articles/204539569-Supported-browsers-for-Participants)?
- Have you updated any relevant documentation or left useful comments in the code?
- Have you reviewed Culture Amp's Web Accessibility guide [Current Compliance, Going Forward, Statement and Answers for Customers](https://cultureamp.atlassian.net/wiki/spaces/Prod/pages/428572998/Web+Accessibility)?
- If this contains substantial visual changes, especially far-reaching ones, have you unlocked the Chromatic step in the pipeline?
- Have Storybook stories been updated?

**If you're new to Kaizen, please ask #prod_design_systems to set up an onboarding session to get you up to speed.** If you have an urgent PR to merge before that happens, it is safest to ask in #prod_design_systems to have a design system advocate review the PR to catch any issues.

### CODE REVIEW

**Comment by katsambarcus** [comment]
@ActuallyACat some of my thoughts:

Where is this navigation bar still being used? This is pretty out of date. Anything with the red enso in the corner is a couple of years old now.

And some minor correction on color names and color values:
 
– Change default navigation bar to be **wisteria-600** on Heart, and keep wisteria-700 on Zen. Correction: **Purple 600** on heart
- Change content navigation bar to be **cluny-200** on Heart, and keep cluny-100 on Zen. Correction: **Blue 100** for Heart

**Comment by ActuallyACat** [comment]
The Navbar app that Unify owns is using Zen Navigation Bar under the hood, and replaces the red Enso with the "product switcher". That means this component needs to be updated before I can update `navigation-bar-app`. There's an example of it in Kaizen here: https://dev.cultureamp.design/ally/heart-navigation/storybook/?path=/story/zennavigationbar-react--with-footer-and-header-components

File: draft-packages/zen-navigation-bar/KaizenDraft/ZenNavigationBar/components/Dropdown.module.scss [hunk 1]
```diff
@@ -8,8 +8,8 @@
   align-items: center;
   position: absolute;
   z-index: $ca-z-index-dropdown;
-  background-color: $kz-color-white;
-  box-shadow: 0 0 ($ca-grid / 2) rgba($kz-color-wisteria-800, 0.3);
+  background-color: $kz-var-color-white;
+  box-shadow: 0 0 ($ca-grid / 2) rgba($kz-var-color-wisteria-800, 0.3);
```
Comment (dmisdm) [inline draft-packages/zen-navigation-bar/KaizenDraft/ZenNavigationBar/components/Dropdown.module.scss:12]: I think the `-rgb-params` suffix is needed on `$kz-var-color-wisteria-800` ?

File: draft-packages/zen-navigation-bar/KaizenDraft/ZenNavigationBar/components/MenuGroup.module.scss [hunk 2]
```diff
@@ -48,13 +48,13 @@
   margin: 0;
   white-space: nowrap;
 
-  color: rgba($kz-color-wisteria-800, 0.8);
+  color: rgba($kz-var-color-wisteria-800, 0.8);
```
Comment (dmisdm) [inline draft-packages/zen-navigation-bar/KaizenDraft/ZenNavigationBar/components/MenuGroup.module.scss:51]: Same here and in this file, need -rgb-params i think!

File: draft-packages/zen-navigation-bar/KaizenDraft/ZenNavigationBar/components/MenuItem.module.scss [hunk 3]
```diff
@@ -34,7 +34,7 @@ $menuItem__margin: 4px;
   @include ca-margin($start: $kz-spacing-md / 2);
 
   display: flex;
-  color: rgba($kz-color-wisteria-800, 0.5);
+  color: rgba($kz-var-color-wisteria-800, 0.5);
```
Comment (dmisdm) [inline draft-packages/zen-navigation-bar/KaizenDraft/ZenNavigationBar/components/MenuItem.module.scss:37]: ^^

File: draft-packages/zen-navigation-bar/KaizenDraft/ZenNavigationBar/helpers/components/Menu.module.scss [hunk 4]
```diff
@@ -9,12 +9,12 @@
   list-style-type: none;
 
   &.final {
-    border-top: 1px solid rgba($kz-color-wisteria-700, 0.1);
+    border-top: 1px solid rgba($kz-var-color-wisteria-700, 0.1);
```
Comment (dmisdm) [inline draft-packages/zen-navigation-bar/KaizenDraft/ZenNavigationBar/helpers/components/Menu.module.scss:12]: ^^

**Review by dmisdm — APPROVED** [review]

**Pull request merged**